src/vcpkg-test/dependencies.cpp

```diff
@@ -215,6 +216,25 @@ struct MockOverlayProvider : IOverlayProvider
 
 static const MockOverlayProvider s_empty_mock_overlay;
 
+static void CHECK_LINES(const LocalizedString& a, StringView b)
```

Comment:
I think plain CHECK would be better here because Catch2 already gives you like a diff view; doing things in a separate function like this robs all context.

Reply:
I would love a diff view -- how do I get it? If I just use `CHECK(a == b)`, I get:

```
with expansion:
  LL"error: version conflict on a:x86-windows: toplevel-spec required 1.0.1 but
  vcpkg could not compare it to the baseline version 1.0.0.
  The two versions used incomparable schemes:
      "1.0.0" was of scheme semver
      "1.0.1" was of scheme string
  This can be resolved by adding an explicit override to the preferred version,
  for example:
      "overrides": [
          { "name": "a", "version": "1.0.0" }
      ]
  See `vcpkg help versioning` for more information."
  ==
  "erro: version conflict on a:x86-windows: toplevel-spec required 1.0.1 but
  vcpkg could not compare it to the baseline version 1.0.0.

The two versions used incomparable schemes:
      "1.0.0" was of scheme semver
      "1.0.1" was of scheme string

This can be resolved by adding an explicit override to the preferred
  version, for example:

    "overrides": [
          { "name": "a", "version": "1.0.0" }
      ]

See `vcpkg help versioning` for more information."
```

Reply:
Maybe I got confused with Mocha..